std_msgs/Header header

 
byte IDLE_IDLE_1 = 0
byte ROBOT_FAIL_1 = 2
byte WALKING_IDLE_1 = 1
byte DOCKING_IDLE = 3
byte IDLE_IDLE_2 =  4
byte IDLE_WORKING = 5
byte BOX_FAIL = 6
byte IDLE_IDLE_3 = 7
byte STANDING_IDLE = 8
byte ROBOT_FAIL_2 = 9
byte WALKING_IDLE_2 = 10
byte SUCCES_IDLE = 11

byte state
